tests/unit/libbpf_test.cpp

```diff
@@ -106,74 +106,54 @@ TEST_CASE("libbpf prog test run", "[libbpf][deprecated]")
     // Bad fd
     REQUIRE(bpf_prog_test_run_opts(nonexistent_fd, &opts) == -EINVAL);
 
-    // NULL data
-    opts.data_in = nullptr;
-    opts.data_size_in = 0;
-    REQUIRE(bpf_prog_test_run_opts(program_fd, &opts) == 0);
-
-    // Zero length data
-    opts.data_in = packet.data();
-    opts.data_size_in = 0;
-    REQUIRE(bpf_prog_test_run_opts(program_fd, &opts) == 0);
-
-    // Context out is too small
-    std::vector<uint8_t> context(1);
-    opts.data_in = packet.data();
-    opts.data_size_in = static_cast<uint32_t>(packet.size());
-    opts.ctx_in = context.data();
-    opts.ctx_size_in = static_cast<uint32_t>(context.size());
-    opts.ctx_out = context.data();
-    opts.ctx_size_out = static_cast<uint32_t>(context.size());
+    // NULL context
```

Comment:
nit: can you fix the comments to have consistent style with the rest of the file (i.e., end in punctuation)?  Yes I know it was already inconsistent before your PR here, so ok if you prefer to leave out of this PR but since you're touching the lines anyway...